homeassistant/components/system_bridge/__init__.py

```diff
@@ -48,10 +49,20 @@
 CONF_TEXT = "text"
 
 SERVICE_OPEN_PATH = "open_path"
+SERVICE_POWER_COMMAND = "power_command"
 SERVICE_OPEN_URL = "open_url"
 SERVICE_SEND_KEYPRESS = "send_keypress"
 SERVICE_SEND_TEXT = "send_text"
 
+POWER_COMMAND_MAP = {
+    "hibernate": "power_hibernate",
+    "lock": "power_lock",
+    "logout": "power_logout",
+    "restart": "power_restart",
+    "shutdown": "power_shutdown",
+    "sleep": "power_sleep",
+}
```

Comment:
Why using key, value pair if only the key is used 

The prefix is handled here `coordinator.websocket_client, f"power_{call.data[CONF_COMMAND]}"`

Reply:
I'm now using this as was originally intended to map the attr based on the key here